web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -982,7 +993,9 @@ function UsageBillingPageInner(props: IUsageBillingPageInnerProps) {
                     {u.name}
                   </td>
                   <td>{u.interactions}</td>
+                  <td>{u.firstActiveDate}</td>
```

Comment:
when firstActiveDate is undefined what does this show / what should it show?

Reply:
Here's what it looks like when those are undefined. I think this is okay.
<img width="743" alt="image" src="https://user-images.githubusercontent.com/1929960/83366790-37c43100-a37f-11ea-8d2f-ec4aa5c10b4a.png">